apps/console/src/app/[entity]/dashboard/page.tsx

```diff
@@ -0,0 +1,22 @@
+import { DashboardPageRender } from "./render";
+import { Metadata } from "next";
+
+export async function generateMetadata(): Promise<Metadata> {
+  try {
+    const metadata: Metadata = {
+      title: `Instill Core | Dashboard`,
+      openGraph: {
+        images: ["/instill-open-graph.png"],
+      },
+    };
+
+    return Promise.resolve(metadata);
+  } catch (error) {
+    console.log(error);
+    return Promise.reject(error);
+  }
+}
```

Comment:
@iamnamananand996 There has no action in the clause that will create error, you can safely remove the catch